event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/resource/RequestValidator.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.services.eventsourcing.source.api.resource;
+
+import static uk.gov.justice.services.eventsourcing.repository.jdbc.Direction.BACKWARD;
+import static uk.gov.justice.services.eventsourcing.repository.jdbc.Direction.FORWARD;
+import static uk.gov.justice.services.eventsourcing.repository.jdbc.FixedPosition.FIRST;
+import static uk.gov.justice.services.eventsourcing.repository.jdbc.FixedPosition.HEAD;
+
+import javax.ws.rs.BadRequestException;
+
+public class RequestValidator {
+    private static final String BAD_HEAD_REQUEST = "Invalid request, cannot request PREVIOUS page from HEAD.";
+    private static final String BAD_FIRST_REQUEST = "Invalid request, cannot request NEXT page from FIRST.";
+
+
+    public static final void validateRequest(final String position, final String direction) {
+        if (HEAD.getPosition().equals(position) && FORWARD.toString().equals(direction)) {
+            throw new BadRequestException(BAD_HEAD_REQUEST);
+        }
+
+        if (FIRST.getPosition().equals(position) && BACKWARD.toString().equals(direction)) {
+            throw new BadRequestException(BAD_HEAD_REQUEST);
+        }
```

Comment:
both using same constant for exception message.

Reply:
sorry